src/SprykerEco/Zed/AvalaraTaxShipmentTax/Business/Mapper/AvalaraLineItemMapper.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\AvalaraTaxShipmentTax\Business\Mapper;
+
+use Generated\Shared\Transfer\AvalaraAddressTransfer;
+use Generated\Shared\Transfer\AvalaraLineItemTransfer;
+use Generated\Shared\Transfer\ShipmentTransfer;
+use SprykerEco\Zed\AvalaraTaxShipmentTax\Dependency\Facade\AvalaraTaxShipmentTaxToMoneyFacadeInterface;
+
+class AvalaraLineItemMapper implements AvalaraLineItemMapperInterface
+{
+    public const SHIPMENT_AVALARA_LINE_TYPE = 'shipment';
```

Comment:
* If it is not used outside a class mark as `protected`.

Reply:
It is used in tax calculator

Reply:
Resolved.